src/RA_Core.cpp

```diff
@@ -872,115 +872,117 @@ API void CCONV _RA_LoadPreferences()
     RA_LOG(__FUNCTION__ " - loading preferences...\n");
 
     SetCurrentDirectory(NativeStr(g_sHomeDir).c_str());
-    FILE* pf = nullptr;
-    fopen_s(&pf, std::string(std::string(RA_PREFERENCES_FILENAME_PREFIX) + g_sClientName + ".cfg").c_str(), "rb");
-    if (pf == nullptr)
+    std::ifstream ifile{ ra::PrefsFilename(), std::ios::binary };
+
+#pragma region Comments
+    /*if (pf == nullptr)
```

Comment:
This situation isn't possible, most of it was commented out already. Deleted my prefs file and it came back.

Reply:
There are several things that generate the prefs file. A successful login is one. Regardless, this functionality is changed with the IConfiguration service in PR 66.

Reply:
Sure, I need to merge the new stuff anyway.